src/components/markdown.js

```diff
@@ -48,15 +48,19 @@ export const newWindowLinkRenderer = (href, title, text) => {
 const SimpleMDE = lazy(() => import('react-simplemde-editor'))
 
 export const MarkdownEditor = props => {
-  return h(Suspense, { fallback: centeredSpinner() }, [h(SimpleMDE, _.merge({
-    options: {
-      autofocus: true,
-      renderingConfig: {
-        singleLineBreaks: false
-      },
-      previewClass: ['editor-preview', 'markdown-body'],
-      previewRender: renderAndSanitizeMarkdown,
-      status: false
-    }
-  }, props))])
+  const editorOptions = useMemo(() => {
+    return _.merge({
+      options: {
+        autofocus: true,
+        renderingConfig: {
+          singleLineBreaks: false
+        },
+        previewClass: ['editor-preview', 'markdown-body'],
+        previewRender: renderAndSanitizeMarkdown,
+        status: false
+      }
+    }, props)
+  }, [])
```

Comment:
I get a warning here about needing to pass in `props`. However, when I do pass in `props` then I see the original bug. I think I don't quite understand how to use `useMemo` and `_.merge` together.

(React Hook useMemo has a missing dependency: 'props'. Either include it or remove the dependency array)

Here is a description of the issue I'm trying to fix:
https://github.com/RIP21/react-simplemde-editor#options

Note that with this code change, the bug goes away in both places where the editor is used. However, I'm noticing that in the data editor usage, if you delete all the text in the description and press Save the deletion is not honored-- it is if you have at least one character left in the editor. I have not yet experimented to see if this bug exists previous to my change.

Reply:
I went back to an old version of dev (before my library upgrade) and verified that the bug where deleting all text in the snapshot data description field pre-existed my work. I will create a separate bug for that.